src/main/java/com/rocana/lucene/codec/v1/RocanaStats.java

```diff
@@ -28,11 +28,36 @@
 import org.apache.lucene.util.IOUtils;
 
 /**
+ * Fork of Lucene's {@link com.rocana.lucene.codec.v1.RocanaStats}.
+ *
+ * Why we forked:
+ *   - To use the other forked classes, like {@link RocanaSegmentTermsEnumFrame}.
+ *
+ * What changed in the fork?
+ *   - Use the other forked classes.
+ *   - Removed trailing whitespace.
+ *   - Changed these javadocs.
+ *
+ * This is one of the forked classes where no logic changed, but to get
+ * the fork to compile we had to fork this class too. That happened with
+ * several classes because they had a hard reference to another class we
+ * forked. Ideally, our forked classes would extend the original Lucene
+ * class and override just the methods we need to change.
+ *
+ * Unfortunately in this particular case, the methods we need to customize
+ * reference a private field and, as this class might be on a critical
+ * performance path, we don't want to pay the performance price of using
+ * reflection to access that private field.
+ *
+ * See ROCANA-8229
```

Comment:
Same comment here about referring to internal only JIRA tickets in open sourced code.